person.rb

```diff
@@ -9,6 +9,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
   def correct_name
```

Comment:
- [ ] Kindly add a  method for adding `rentals`. This fulfills the "person has many rentals:".

### Instructions:
![image](https://github.com/ybadr99/school_library/assets/50990925/a72da01d-7bbc-4a68-870d-c098aa24ed24)